README.md

```diff
@@ -112,19 +112,25 @@ $ ./packages/percolatestudio-migrations/migrate.sh latest --settings ./setting.j
 
   Migrations set a lock when they are migrating, to prevent multiple instances to prevent multiple instances of your clustered app from running migrations simultaneously. If you migrations throw an exception, you will need to manually remove the lock (and ensure your db is still consistent) before re-running the migration. Update the migrations collection like this:
 
-  ``` sh
-  meteor mongo
+  ```
+  $ meteor mongo
+
   db.migrations.update({_id:"control"}, {$set:{"locked":false}});
   exit
   ```
 
 ## Contributing
 
-Write some code. Write some tests. To run the tests, do:
+1. Write some code.
+2. Write some tests.
+3. From this package's local directory, start the test runner:
+
+    ```
```

Comment:
FWIW, I chose not to use the `sh` language indicator on this fenced code block as it makes "test" get inappropriate syntax highlighting.